src/iceberg/table_metadata.h

```diff
@@ -124,6 +124,12 @@ struct ICEBERG_EXPORT TableMetadata {
   /// A `long` higher than all assigned row IDs
   int64_t next_row_id;
 
+  static Result<std::unique_ptr<TableMetadata>> Make(
+      const iceberg::Schema& schema, const iceberg::PartitionSpec& spec,
```

Comment:
Do you think we should rename functions below to `GetSchema`, `GetSnapshot`, etc. so we don't need to write `iceberg::` prefix here and elsewhere?